app/routes/image-set.js

```diff
@@ -3,6 +3,17 @@ import OrganizationRouteMixin from 'lion-guardians/mixins/organization-route';
 import RequireUserMixin from 'lion-guardians/mixins/require-user';
 
 export default Ember.Route.extend( OrganizationRouteMixin, RequireUserMixin, {
+  beforeModel: function() {
+    var route = this;
+    return this._super().then(function() {
+      if (window.SKIP_MAPS) {
+        return Ember.RSVP.resolve();
```

Comment:
to clarify: you do not need to return `Ember.RSVP.resolve()` here -- you can do nothing if you are not loading the map. You *do* need to return `route.loadGoogleMaps()` if you are loading the map, though.